proxy_init/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:xenial
+
+ADD ./proxy_init.sh /proxy_init.sh
+RUN chmod 755 /proxy_init.sh
```

Comment:
I think I've read somewhere you can do this now:
```suggestion
ADD --chmod=0755 ./proxy_init.sh /proxy_init.sh
```

But I might be wrong 🤔

Reply:
https://docs.docker.com/engine/reference/builder/#add

I think you are mixed up with `--chown`?